roles/upload-site/tasks/main.yml

```diff
@@ -41,32 +41,40 @@
 # per-site basis. That will allow us to move past errors in this task, and run
 # "drush updb" a second time in the following task.
 - name: Update database schema with drush updb
-  shell: "{{drush_alias }} {{ item }}"
+  shell: "{{ drush_alias }} {{ item }}"
   with_items:
     - "-y updb"
   ignore_errors: "{% if ignore_updb_errors == 'TRUE' %}yes{% endif %}"
 
+# This only runs "drush rr" if ignore_updb_errors == "TRUE"
+- name: Run drush rr before running drush updb a second time for recalcitrant sites
+  shell: "{{ drush_alias }} {{ item }}"
+  with_items:
+    - "-y rr"
+  when: ignore_updb_errors == "TRUE"
+  ignore_errors: yes
+
 # This only runs "drush updb" if ignore_updb_errors == "TRUE"
 - name: Run drush updb a second time for recalcitrant sites
-  shell: "{{drush_alias }} {{ item }}"
+  shell: "{{ drush_alias }} {{ item }}"
   with_items:
     - "-y updb"
   when: ignore_updb_errors == "TRUE"
 
 - name: Set site up as a Department site
-  shell: "{{drush_alias }} {{ item }}"
+  shell: "{{ drush_alias }} {{ item }}"
   with_items:
-    - "-y vset install_profile stanford_dept"
+    - "-y vset install_profile stanford_sites_jumpstart"
```

Comment:
How come jumpstart? Also, it is time to make this dynamic?

Reply:
Damn these ham hands! I didn’t mean to commit this.  And yes it is, but I got distracted by something shiny.